osu.Game/Localisation/SkinEditorStrings.cs

```diff
@@ -42,7 +42,17 @@ public static class SkinEditorStrings
         /// <summary>
         /// "Currently editing"
         /// </summary>
-        public static LocalisableString CurrentlyEditing => new TranslatableString(getKey(@"currently_editing"), "Currently editing");
+        public static LocalisableString CurrentlyEditing => new TranslatableString(getKey(@"currently_editing"), @"Currently editing");
+
+        /// <summary>
+        /// "Revert to default?"
+        /// </summary>
+        public static LocalisableString RevertToDefault => new TranslatableString(getKey(@"revert_to_default"), @"Revert to default?");
+
+        /// <summary>
+        /// "The skin will return to the state it was in upon import"
+        /// </summary>
+        public static LocalisableString ResetDialogue => new TranslatableString(getKey(@"the_skin_will_return_to"), @"The skin will return to the state it was in upon import");
```

Comment:
all this localisation does is add a question mark compared to the existing common revert to default, opted for doing it this way since i'm not sure if all languages we support use question marks, so concatenating the string might've not been ideal?

Reply:
You left this comment on the wrong localisation string. But yes, it is probably wise to not append a question mark manually. Some languages will have extra punctuation rules concerning questions (see: French which routinely places a space between the sentence and the question mark, or Spanish, which does the entire `¿question?` thing).

Reply:
Call this `RevertToDefaultDescription` or something. And it should end with a full stop.